### ISSUE TITLE
Different media type returned by endpoint on error produces 406

### ISSUE THREAD

**Issue by mivr83** [swagger-api/swagger-ui#6196]
### Q&A (please complete the following information)
 - OS: Windows
 - Browser: Chrome
 - Version: 83.0.4103.116 
 - Method of installation: springdoc-openapi
 - Swagger-UI version: latest?
 - Swagger/OpenAPI version: OpenAPI 3.0

#### The problem
I have endpoint which produces two media types: `multipart/form-data` or `application/json`.
Multipart is returned on 200/ok and json on error (404, 400 etc.). When i configure open api (through annotations in spring boot app by springdoc library) i get this result in swagger ui: i get dropdown list of returned mediatypes and selected valu is used in `curl` call and here comes catch, i dont know at which time i get error and thus i receive json instead of multipart media and in result i get 406 instead of 404 for example.

#### What would i expect
I would expect that there is option to tell swagger ui to expect multiple produced media types without need to selec exact one so it makes `curl` call with parameter `-H "accept: multipart/form-data, application/json"`. This way app will produce one of expected types and it get correctly displayed. 
In specification are defined mutiple responses with json media type (error ones) and on with multipart media type so it make sense that app call should accept both of this types without need.
**So basically what shoud be done is: get selected media type from 200/ok and add all other distinct media types from other responses.**

Is there any way to achieve this with current version of swagger ui (it is possible but i have missed something) or should i submit feature request?

here is issue in springdoc project: https://github.com/springdoc/springdoc-openapi/issues/760

Thx

**Comment by hkosova** [comment]
Thanks for the report, @mivr83! This was previously reported in #5649 so I'll close this one as a duplicate.

See also this comment: https://github.com/swagger-api/swagger-ui/issues/5649#issuecomment-543355710
